test-case-reporting/index.ts

```diff
@@ -55,16 +55,23 @@ app.get('/test-results/build/:buildNumber', async (req, res) => {
 
 app.get('/test-results/history/:testCaseId', async (req, res) => {
   const {testCaseId} = req.params;
-  const {json} = req.query;
+  const {json, limitStr, offsetStr} = req.query;
```

Comment:
The query params should just be `limit` and `offset`. The name collision will be annoying, but it keeps query params straightforward

Reply:
Done naming the query params `limit` and `offset`; renamed the parsed versions to `limitNum` and `offsetNum` (we can't set `limit` to the parsed number unless we make it type `any`)